docs/troubleshooting.md

```diff
@@ -121,6 +122,15 @@ Some of the common reasons we have encountered that caused this `MESSAGE` are qu
 
 If you are trying to [install an operator from the cli](https://github.com/operator-framework/operator-marketplace#installing-an-operator-using-marketplace), the packages listed in the CatalogSourceConfig must have valid package names. If you have private app-resgistries, please follow the steps listed [here](https://github.com/operator-framework/operator-marketplace/blob/master/docs/how-to-authenticate-private-repositories.md).
 
+## Conflicting Package Names
+
+The Marketplace operator does not support reusing package names between two OperatorSources. To avoid unexpected behavior users should avoid adding any OperatorSources that reuse existing package names.
```

Comment:
I would rather say that Package names are global across OperatorSources. You should also mention what will happen if there are conflicting package names.

Reply:
I actually think it is better to avoid describing the particular behavior and issues associated with multiple package names -- if we do, and that behavior changes, then we have to update this doc to describe a different bug. Why not leave it as simply "marketplace doesn't currently support multiple operator sources with the same package names" ? What value do we get by describing the specific limitations?

Reply:
Aren't both statements saying saying the same thing i.e. describing the specific limitations? Or are you saying that we don't mention what happens when there are conflicts? It is useful to describe that here because I can see that happening and users wondering what is going on. Moreover documentation is for describing current behavior. And yes, when that behavior changes we need to update the documentation.

Reply:
I just think in this specific case what we are describing is a possibly non deterministic behavior (because in some cases it will succeed if the in memory cache happens to respond just the right way) and describing it in that detail may leave users more confused than simply stating that it can cause unexpected behavior.

Reply:
I am fine with adding a line explicitly stating that it can cause unexpected behavior.

Reply:
> I am fine with adding a line explicitly stating that it can cause unexpected behavior.

Isn't this already accomplished with the existing line: 
> To avoid unexpected behavior users should avoid adding any OperatorSources that reuse existing package names.

Reply:
You are right. I take back my request. I would slightly modify the statement though. `To avoid unexpected behavior users should avoid adding any OperatorSources that contain existing packages`

Reply:
Wonderful - I will use your suggested wording, thanks for the review.